aten/src/ATen/native/cuda/TensorCompare.cu

```diff
@@ -30,9 +34,47 @@ void where_kernel_impl(TensorIterator &iter, ScalarType condition_type) {
   });
 }
 
+void isposinf_kernel_impl(TensorIterator &iter) {
+  if (c10::isIntegralType(iter.input_dtype(), /*include_bool=*/true)) {
```

Comment:
With the isfinite I think you can just dispatch on the float types here, too.

Reply:
addressed